core/src/components/toggle/toggle.tsx

```diff
@@ -337,6 +396,7 @@ export class Toggle implements ComponentInterface {
             part="label"
           >
             <slot></slot>
+            {this.renderHintText()}
```

Comment:
The text (helper or error) is being read twice by VoiceOver on Safari, Firefox, and Chrome. After some quick debugging, it seems that this happening because the text is inside of label. ion-input has it after the <label> which mimics how the native input states it.

https://github.com/user-attachments/assets/a50e2d91-36db-4eca-8d93-d42124f21657